polaris.shopify.com/pages/api/v0/tokens/[tokens].tsx

```diff
@@ -97,67 +60,46 @@ const formatTokenGroup = (tokenGroup: TokenGroup, format: Format) => {
       .join('\n');
   }
 
-  return tokenGroup;
+  return tokenValues;
 };
 
 const handler = async (req: NextApiRequest, res: NextApiResponse) => {
-  const formatParam = (req.query.format || 'json') as Format;
-  const schemeParam = (req.query.scheme || 'light') as Scheme;
+  const formatParam = isFormat(req.query.format) ? req.query.format : 'json';
+  const schemeParam = isScheme(req.query.scheme) ? req.query.scheme : 'light';
 
   if (typeof formatParam === 'string' && typeof schemeParam === 'string') {
-    const colorSchemeTokenGroup = colorSchemeMap[schemeParam];
-    const tokenGroupParam: any = req.query.tokens || '';
-    let tokenGroups: string[] = [];
+    const tokenGroupParam = (req.query.tokens || '') as TokenGroupKey;
+    let tokenData: TokenGroup = {};
 
     // Determine which list(s) we are querying for based on the token param
     if (tokenGroupParam === 'all') {
-      tokenGroups = tokenGroups.concat(staticTokenGroupKeys);
-    }
+      staticTokenGroupKeys.forEach((group) => {
+        const tokenGroup: TokenGroup = tokens[group] || {};
```

Comment:
```suggestion
        const tokenGroup = tokens[group];
```
Do we need the fallback or the explicit type?